docker/Dockerfile.ci_cpu

```diff
@@ -124,15 +124,6 @@ RUN bash /install/ubuntu_install_androidsdk.sh
 ENV ANDROID_HOME=/opt/android-sdk-linux/
 ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147/
 
-# Install CMSIS_NN
-COPY install/ubuntu_install_cmsis.sh /install/ubuntu_install_cmsis.sh
-RUN bash /install/ubuntu_install_cmsis.sh /opt/arm/ethosu/cmsis
-ENV CMSIS_PATH=/opt/arm/ethosu/cmsis/
-
-# Arm(R) Ethos(TM)-U NPU driver
-COPY install/ubuntu_install_ethosu_driver_stack.sh /install/ubuntu_install_ethosu_driver_stack.sh
```

Comment:
run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-contrib-test_ethosu tests/python/contrib/test_ethosu -n auto

we need this line moved alongside as well.

Reply:
we should move that after we updated the qemu image with the dependencies for this test

Reply:
I can update this PR by adding `Arm(R) Ethos(TM)-U NPU driver` to the qemu, we merge this. Then we will update the qemu image and then send another PR to remove `test_ethosu` and move it to qemu.

Reply:
sorry we already have that. I meant `Github Arm(R) Ethos(TM)-N NPU driver`, I believe that is required, correct me if I'm wrong.

Reply:
I tested it in qemu, looks fine. So I moved it